test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/ExtendedQueryTagTests.cs

```diff
@@ -42,11 +44,7 @@ public ExtendedQueryTagTests(HttpIntegrationTestFixture<Startup> fixture)
         // [Trait("Category", "bvt")] // TODO: Enable once functions are enabled in PAAS
         public async Task GivenExtendedQueryTag_WhenReindexing_ThenShouldSucceed()
         {
-            if (_isUsingInProcTestServer)
-            {
-                // AzureFunction doesn't have InProc test sever, skip this test.
-                return;
-            }
+            await using IAsyncDisposable hostExecution = await _functionApp.StartAsync();
```

Comment:
There are still some issues running both tests back-to-back. I need to figure out what's up with the host.